.github/workflows/integration-test-call.yml

```
@@ -45,6 +49,13 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.BOT_TOKEN }}
 
+      - name: Add success label to pull request
+        if: ${{ needs.query.result == 'success' && github.event.inputs.pr_number != '' }}
+        run: |
+          gh pr edit "${{ github.event.inputs.pr_number }}" --add-label "integration tests succeeded"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Minor: this is what we've tended to do in our other workflows:

```suggestion
        run: |
          gh pr edit "$PR_NUM" --add-label "integration tests succeeded"
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          PR_NUM: ${{ github.event.inputs.pr_number }}
```

But, what you have is fine.